### PR TITLE
[NLP-7736]: considering question words as stopwords in all export formats

### PR DESCRIPTION
**Pull request by satyaad-kore**

### CODE REVIEW

File: request_type/CSVExportParser.py [hunk 1]
```diff
@@ -29,11 +26,18 @@ def parse(self):
             self.print_verbose(error_msg)
 
     def get_stopwords_from_csv(self, faq_row_counter):
-        for row in self.faq_payload[faq_row_counter + 1:]:
-            if row[0] == 'kgParams':
-                return set(row[4:])
-            else:
-                return self.get_stopwords()
+        try:
+            for row in self.faq_payload[faq_row_counter + 1:]:
```
Comment (surendra-kore-zz) [inline request_type/CSVExportParser.py:32]: if kg_param row is not present

File: request_type/CSVExportParser.py [hunk 2]
```diff
@@ -1,9 +1,6 @@
-import traceback
 from itertools import count as it_count
 from collections import namedtuple
-
-from strategy.phrase_finder import PhraseFinder
-from request_type.Parser import Parser
+from request_type.Parser import *
 from log.Logger import Logger
 from common import *
```
Comment (surendra-kore-zz) [inline request_type/CSVExportParser.py]: please refactor

**Review by surendra-kore-zz — CHANGES_REQUESTED** [review]

File: request_type/CSVExportParser.py (hunk 1 above)
Comment (satyaad-kore) [reply request_type/CSVExportParser.py:32]: kept that outside loop

File: request_type/CSVExportParser.py (hunk 2 above)
Comment (satyaad-kore) [reply request_type/CSVExportParser.py]: done

**Review by surendra-kore-zz — APPROVED** [review]

**Pull request merged**